system-tests/projects/migration-e2e-custom-integration-with-projectId/README.md

```diff
@@ -0,0 +1,35 @@
+## Migration E2E Custom Integration with projectId
+
+An e2e project with a custom `integrationFolder` named `src` and projectId. It uses the default `testFiles`. We will keep the custom `intergrationFolder`, but it'll be part of `e2e.specPattern`. We will rename their specs to use the `.cy.js` extension.
```

Comment:
Don't forget to update this to explain this fixture - it  says "We will rename their specs to use the `.cy.js` extension", which isn't right.

In this case, "this project has as `projectId` which means we do not migrate to the new `.cy.js` extension, because this will interfere with the historical data on Cypress Cloud" or something like that.